bin/boot_docker

```diff
@@ -25,6 +26,10 @@ $0 [OPTION]
 -P VALUE	set the SENDGRID_PASSWORD variable (used for emails)
 -o VALUE	set the OAUTH_CLIENT_ID variable (for Google OAuth2)
 -S VALUE	set the OAUTH_CLIENT_SECRET variable (for Google OAuth2)
+-y VALUE	set the SENTRY_DSN variable (for Sentry error reporting, no default)
+-g VALUE  set the GA_TRACKING_ID variable for tracking usage via Google Analytics (no default)
+-w VALUE  set the TCELL_APP_ID variable (for integrating with TCell web application firewall, if enabled)
+-W VALUE  set the TCELL_API_KEY variable (for integrating with TCell web application firewall, if enabled)
```

Comment:
Whitespace inconsistency here -- it seems spaces were used instead of tabs.

Reply:
I'll check to see what the majority are and conform to that.